apps/home/stylesheets/quotes.styl

```diff
@@ -0,0 +1,33 @@
+bp = 1024px
+
+.HomeQuotes
+  margin-bottom 6em
+
+  &__cta
+    margin-bottom 2em
+
+  &__quote
+    display flex
+    flex-direction row
+    width 100%
+    margin 3em 0
+    +below(bp)
+      padding 0 2em
+
+  &__icon
+    width 120px
+    margin-right 1em
+    text-align center
+
+    > img 
+      width 50%
+
+  &__copy
+    font-family 'Courier New', Courier, monospace
+    font-weight bold
+    font-size font-size-sm
+    line-height line-height-tall
+    max-width 475px
+    > span
+      background-color rgb(255, 244, 107)
```

Comment:
What's this color doing?

Reply:
Can we extract it into colors?

Reply:
Highlight
![screen shot 2018-02-26 at 5 17 47 pm](https://user-images.githubusercontent.com/821469/36698851-fbfdbdc6-1b18-11e8-9c59-d39e3884cebe.png)

Reply:
And sure